server.js

```diff
@@ -1,6 +1,9 @@
-import app from './src/app';
+import application from './src/app';
 const port = 3000;
 
-app.listen(port, () => {
-  console.log(`app running on port ${port}`);
-});
+application()
```

Comment:
Looking at the code it seems app.js's responsability is to expose a function that does an express instance  setup after connecting to MongoDB and returns it to the function caller. 

Therefore as a **personal preference** i'd name this `setupApp` (or similar `action` name) and give it an instance of express from outside (from server.js). E.g: `setupApp(express()).then()...`.